features.json

```
@@ -124,6 +124,22 @@
 				"simd": "--enable-simd",
 				"threads": "--enable-threads"
 			}
+		},
+		"Node.js": {
+			"url": "https://nodejs.org/",
+			"logo": "/images/nodejs.svg",
+			"version": "12.18.1",
```

Let's omit the non-important patch version, like in other engines/browsers. It shouldn't ever affect supported feature set, and only makes output a bit more verbose.

```suggestion
			"version": "12.18",
```